bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-notifications.js

```diff
@@ -89,12 +90,417 @@ export default function defineOHBlocks_Notifications (f7, isGraalJs) {
+
+  Blockly.Blocks['oh_sendCloudNotification'] = {
+    init: function () {
+      const userImageField = new Blockly.FieldImage(usersImage, 15, 15, undefined, this.onClickUser)
+      userImageField.setTooltip('Comma separated users. If not provided it results into a broadcast')
+      const iconImageField = new Blockly.FieldImage(iconImage, 15, 15, undefined, this.onClickIcon)
+      iconImageField.setTooltip('Icon name')
+      const tagImageField = new Blockly.FieldImage(tagImage, 15, 15, undefined, this.onClickTag)
+      tagImageField.setTooltip('Tag')
+      const headerTitleImageField = new Blockly.FieldImage(headerTitleImage, 15, 15, undefined, this.onClickHeaderTitle)
+      headerTitleImageField.setTooltip('Header title of Notification')
+      const referenceImageField = new Blockly.FieldImage(referenceImage, 15, 15, undefined, this.onClickReference)
+      referenceImageField.setTooltip('Reference of the message')
+      const actionImageField = new Blockly.FieldImage(actionImage, 15, 15, undefined, this.onClickAction)
+      actionImageField.setTooltip('Action command to be sent')
+      const actionButton1ImageField = new Blockly.FieldImage(actionButton1Image, 15, 15, undefined, this.onClickActionButton1)
+      actionButton1ImageField.setTooltip('Action Button 1 command and on-click action')
+      const actionButton2ImageField = new Blockly.FieldImage(actionButton2Image, 15, 15, undefined, this.onClickActionButton2)
+      actionButton2ImageField.setTooltip('Action Button 2 command and on-click action')
+      const actionButton3ImageField = new Blockly.FieldImage(actionButton3Image, 15, 15, undefined, this.onClickActionButton3)
+      actionButton3ImageField.setTooltip('Action Button 3 command and on-click action')
+      const mediaImageField = new Blockly.FieldImage(mediaImage, 15, 15, undefined, this.onClickMedia)
+      mediaImageField.setTooltip('A Media URL that can be clicked and can be accessed publicly to be shown with the notification')
+
+      this.appendValueInput('message')
+        .setCheck('String')
+        .appendField(userImageField, 'imgUsers')
+        .appendField(headerTitleImageField, 'headerTitleIcon')
+        .appendField(iconImageField, 'imgIcon')
+        .appendField(tagImageField, 'tagIcon')
+        .appendField(referenceImageField, 'tagIcon')
+        .appendField(mediaImageField, 'mediaIcon')
+        .appendField(actionImageField, 'tagIcon')
+        .appendField(actionButton1ImageField, 'actionButton1Icon')
+        .appendField(actionButton2ImageField, 'actionButton2Icon')
+        .appendField(actionButton3ImageField, 'actionButton3Icon')
+        .appendField(new Blockly.FieldDropdown([['send notification', 'cloud'], ['send log', 'log']]), 'type')
```

Comment:
I think this naming is better (though it is only internal naming, but it makes more sense this way IMO):

```suggestion
        .appendField(new Blockly.FieldDropdown([['send notification', 'push'], ['send log', 'log']]), 'type')
```